### PR TITLE
Make first-paint optional

### PR DESCRIPTION
**Pull request by noamr**
How is this @npm1 @yoavweiss?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/paint-timing/pull/81.html" title="Last updated on Apr 1, 2020, 6:54 PM UTC (2eeedbb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/paint-timing/81/e85cc0a...2eeedbb.html" title="Last updated on Apr 1, 2020, 6:54 PM UTC (2eeedbb)">Diff</a>

### CODE REVIEW

File: painttiming.bs [hunk 1]
```diff
@@ -192,7 +192,7 @@ Reporting paint timing {#sec-reporting-paint-timing}
     When asked to [=mark paint timing=] given a [=Document=] |document| as input, perform the following steps:
 
     1. Let |paintTimestamp| be the [=current high resolution time=].
-    1. If this instance of [=update the rendering=] is the [=first paint=] of |document|, then invoke the [[#report-paint-timing]] algorithm with |document|, <code>"first-paint"</code>, and |paintTimestamp|.
+    1. If this instance of [=update the rendering=] is the [=first paint=] of |document|, and the user-agent is configured to mark [=first paint=], then invoke the [[#report-paint-timing]] algorithm with |document|, <code>"first-paint"</code>, and |paintTimestamp|.
```
Comment (yoavweiss) [inline painttiming.bs]: Dunno is "is configured" is the right term here. Maybe "supports"?

Comment (noamr) [reply painttiming.bs]: I borrowed it from https://html.spec.whatwg.org/multipage/browsers.html#the-rules-for-choosing-a-browsing-context-given-a-browsing-context-name. 

Either works for me... WDYT?

Comment (yoavweiss) [reply painttiming.bs]: Interesting. I'm cool with it if @rniwa is

Comment (npm1) [inline painttiming.bs]: nit: s/user-agent/user agent

File: painttiming.bs [hunk 2]
```diff
@@ -212,6 +212,9 @@ Reporting paint timing {#sec-reporting-paint-timing}
         NOTE: A parent frame should not be aware of the paint events from its child iframes, and vice versa. This means that a frame that contains just iframes will have [=first paint=] (due to the enclosing boxes of the iframes) but no [=first contentful paint=].
 
         NOTE: A [=document=] is not guaranteed to mark [=first paint=] [=first contentful paint=]. A completely blank [=document=] may never mark [=first paint=], and a [=document=] containing only elements that are not [=contentful=], may never mark [=first contentful paint=].
+
+    NOTE: The marking of [=first paint=] is optional. User-agents implementing paint timing should at the very least mark [=first contentful paint=] .
```
Comment (npm1) [inline painttiming.bs:216]: nit: user agents, not user-agents.

Comment (noamr) [reply painttiming.bs:216]: Fixed

**Pull request merged**